Rakefile

```diff
@@ -37,3 +37,10 @@ Rake::RDocTask.new do |rdoc|
   rdoc.rdoc_files.include('README*')
   rdoc.rdoc_files.include('lib/**/*.rb')
 end
+
+task :docker do
+  cmd = "docker-compose up -d && docker exec -i -t shopify_api bash"
+  unless system(cmd, err: File::NULL)
+    abort("Something went wrong, do you have Docker and Docker-Compose installed?")
```

Comment:
I think there could be other reasons for this failing (lack of internet connection, some kind of internal docker error) so let's not swallow the error and just print it. It's less clean but it will help with debugging.

Reply:
@tylerball What do you suggest here?

Reply:
If you just use `exec` instead of `system` it will raise any errors up to the user and you don't need to rescue anything

Reply:
Sweet, I didn't know that. Thnx